packages/vite/src/node/ssr/ssrManifestPlugin.ts

```diff
@@ -40,6 +44,51 @@ export function ssrManifestPlugin(config: ResolvedConfig): Plugin {
               })
             }
           }
+          if (chunk.code.indexOf(preloadMethod) > -1) {
+            // generate css deps map
+            const code = chunk.code
+            let imports: ImportSpecifier[]
+            try {
+              imports = parseImports(code)[0].filter((i) => i.d > -1)
+            } catch (e: any) {
+              this.error(e, e.idx)
+            }
+            if (imports.length) {
+              for (let index = 0; index < imports.length; index++) {
+                const { s: start, e: end, n: name } = imports[index]
+                // check the chunk being imported
+                const url = code.slice(start, end)
+                const deps: string[] = []
+                // compatible target = esnext
+                if ((url[0] === `"` || url[0] === `'`) && (url[url.length - 1] === `"` || url[url.length - 1] === `'`)) {
```

Comment:
Maybe we should use something more safe like this:

https://github.com/prettier/plugin-pug/blob/a81340d5d3be0c2c54ce046f8305f441ec300a22/src/utils/common.ts#L167-L174

Reply:
It's ok, also change [there]( https://github.com/vitejs/vite/blob/main/packages/vite/src/node/plugins/importAnalysisBuild.ts#L243,)  by the way?